app/components/school_comparison_component/school_comparison_component.html.erb

```diff
@@ -1,6 +1,4 @@
-<div
-  id="<%= @id %>"
-  class="school-comparison-component-callout-row row mt-2">
```

Comment:
This breaks the layout as the naming convention here is different (can't remember why) so the styles in the component need updating. See, e.g. baseload insights page. 

The group version is OK I think as the `-callout-row` class is not on the outer element.

Highlights benefit of previews as sometimes easier to check the previews rather than the pages where component is being used. Don't think we have them for these components.

Reply:
This is a good spot, thanks. I've wrapped the component in a div to put the classname in, and renamed the classes used to have more concise names, scoped to this classname. Hope this is ok.

I've also added basic previews for this and the school group comparison component